C++/Indentation Rules.tmPreferences

```diff
@@ -23,7 +23,7 @@
 		<key>bracketIndentNextLinePattern</key>
 		<string>(?x)
 		^ \s* \b(if|while|else)\b [^;]* $
-		| ^ \s* \b(for)\b .* $
+		|   ^ \s* \bfor\b (?:[^;]++|;[ ]+(?!$))+ \s* $
```

Comment:
This pattern breaks indention after

```C
for(i=0;dd<5;i++)
```

The rule shouldn't require a whitespace after each `;` to make indention work properly.